parsers/ENTSOE.py

```diff
@@ -410,9 +407,7 @@
         "required": ["coal"],
         "expected_range": (5000, 50000),
     },
-    "LT": {
-        "required": ["biomass", "gas", "hydro", "solar", "wind"]
```

Comment:
Huh, that is weird. Does this mean that gas was already required? Is there a way to set it so that it must be above 0?

Reply:
I think you accidentally just reviewed the formatting commit 😅
It was not required before.

Huh there might be, we our own validate lib and the comment in there says:
```
        expected_range: tuple | dict
            Checks production total against expected range.
            Tuple is in form (low threshold, high threshold), e.g. (1800, 12000).
            If a dict, it should be in the form
            {
            'nuclear': (low, high),
            'coal': (low, high),
            }
```
So I'll change it to the dict based version of the threshold and test it out.
What should the minimum threshold be in your opinion?